README.md

```diff
@@ -1,5 +1,35 @@
 # TranslateX
-An Android App based on NLP(Natural Language processing) to translate text from one language to another
-
-It will use Firbase ML Kit's "On Device Translation"
+## Table of Contents
+Getting Started
+Objective
+Creating issues
+Already created issues
+Labels
+App icon
+Deploying bots
+Resources
+Rules and Scoring System
+## Getting Started
```

Comment:
In this "Getting Started" section can you also include about setting up the development environment. It really helps contributors to have a smooth start if they want to contribute to the project.
Also mention about the prerequisites if any.

Reply:
I have made the changes,can you please review it?